docs/setup.md

```diff
@@ -24,26 +24,19 @@ For optimal performance of the **Outbreak Response Framework** solution pack, in
 
-- **Fortinet FortiGate** - Fortinet FortiGate enterprise firewall provides high performance, consolidated advanced security, and granular visibility for broad protection across the entire digital attack surface. To configure and use the Fortinet FortiGate connector, refer to [Configuring Fortinet FortiGate](https://docs.fortinet.com/fortisoar/connectors/fortigate_firewall)
-
-- **Jira** - Jira is a widely used issue-tracking and project management software developed by Atlassian. It helps teams collaborate, organize, and track their work, tasks, and projects. With features like customizable workflows, agile project management, and extensive reporting capabilities, Jira facilitates efficient project planning, execution, and monitoring. To configure and use the Jira connector, refer to [Configuring Jira](https://docs.fortinet.com/fortisoar/connectors/)
-
 - **NIST National Vulnerability Database** - The NIST National Vulnerability Database (NVD) is the U.S. government repository of standards-based vulnerability management data represented using the Security Content Automation Protocol (SCAP). This data enables automation of vulnerability management, security measurement, and compliance. The NVD includes databases of security checklist references, security-related software flaws, misconfigurations, product names, and impact metrics. To configure and use the NIST National Vulnerability Database connector, refer to [Configuring NIST National Vulnerability Database](https://docs.fortinet.com/fortisoar/connectors/nist-nvd)
 
-- **ServiceNow** - ServiceNow connector provides functionality to create, read, update, and delete records of type Table and Catalog. To configure and use the ServiceNow connector, refer to [Configuring ServiceNow](https://docs.fortinet.com/fortisoar/connectors/servicenow-v2-0-1)
 - **Splunk** - Splunk connector allows users to invoke search, fetch events to related search, invoke alert actions, update notables, sync Splunk users to FortiSOAR, etc. To configure and use the Splunk connector, refer to [Configuring Splunk](https://docs.fortinet.com/fortisoar/connectors/splunk_new)
 
 ## Setup Outbreak Response Framework on FortiSOAR
 
-After installation of the **Outbreak Response Framework** solution pack, run the configuration wizard to ready your FortiSOAR environment to investigate Outbreak Alerts. This wizard helps you select the configured "Threat Detection Integrations" and "Ticketing/ITSM Integrations" on FortiSOAR.
+After installation of the **Outbreak Response Framework** solution pack, run the configuration wizard to ready your FortiSOAR environment to investigate Outbreak Alerts. This wizard helps you select the configured **Threat Detection Integrations** on FortiSOAR.
```

Comment:
This wizard helps you select the configured **Threat Detection Integrations** on FortiSOAR. ---> This wizard assists you in selecting and configuring the **Threat Detection Integrations** on FortiSOAR.